src/partials/footer.html

```diff
@@ -1 +1,46 @@
-<footer class="footer"></footer>
+<footer class="footer">
+  <div class="container">
+    <div class="footer-tumb">
+      <div class="footer-tegline">

```

Comment:
Between `.footer-tegline` and `.footer-social` should be a margin, not a padding from one of them.
When we need push apart sibling elements, we need to use margin property exactly.